packages/react-storybook-addon/src/hooks.ts

```diff
@@ -18,3 +19,11 @@ export interface FluentGlobals extends StorybookArgs {
 export function useGlobals(): [FluentGlobals, (newGlobals: FluentGlobals) => void] {
   return useStorybookGlobals();
 }
+
+/**
+ * Uses storybook addon API to update globals with the correc theme Id
+ * Can be used externally be decorators and other addon-docs containers
+ */
+export function setGlobalTheme(themeId: ThemeIds): void {
```

Comment:
just wondering, as we are already exporting `theme` etc from this package, shouldn't this be handled rather in react-components thus removing from api surface of this component ?

Reply:
but then how would we set the storybook global ? we would need to consume `addons.getChannel` from `react-components` and use the internal API

Reply:
at this point I think it;s okay to have this line `addons.getChannel().emit('updateGlobals', { globals: { [THEME_ID]: themeId } });` in react components ( those theme constants are already  exposed )

Reply:
Moved `setGlobalTheme` directly to the `ThemePicker` compoenent that uses it